site/en/contribute/breaking-changes.md

```diff
@@ -71,52 +67,57 @@ should contain the URL of the GitHub issue. As the flag name starts with
 
 In the commit description, add a brief summary of the flag.
 Also add [`RELNOTES:`](release-notes.md) in the following form:
-`RELNOTES: --incompatible_name_of_flag has been added. See #yxz for details`
+`RELNOTES: --incompatible_name_of_flag has been added. See #xyz for details`
 
 The commit should also update the relevant documentation. As the documentation
 is versioned, we recommend updating the documentation in the same commit
 as the code change.
 
-
 ## Labels {:#labels}
 
-Once the commit is merged, add the label
+Once the commit is merged and the incompatible change is ready to be adopted, add the label
 [`migration-ready`](https://github.com/bazelbuild/bazel/labels/migration-ready){: .external}
 to the GitHub issue.
 
-Later a [Bazel release manager](https://github.com/bazelbuild/continuous-integration/blob/master/docs/release-playbook.md){: .external}
-will update the issue and replace the label with `migration-xx.yy`.
-
-The label `breaking-change-xx.yy` communicates when we plan to flip the flag. If
-a migration window needs to be extended, the author updates the label on GitHub
-issue accordingly.
-
 If a problem is found with the flag and users are not expected to migrate yet:
-remove the flags `migration-xx.yy`.
+remove the flags `migration-ready`.
```

Comment:
Making people look at the issue to decide if it is migration ready or not it not going to be a great experience.  A retraction of a plan to change behavior has to be acompannied by an announcmented on bazel-discuss that we are changing the plan.

Reply:
> Making people look at the issue to decide if it is migration ready or not it not going to be a great experience.

Why? I think it's pretty easy to check what flags are migration ready by just opening [this link](https://github.com/bazelbuild/bazel/issues?q=is%3Aopen+is%3Aissue+label%3Amigration-ready) instead of going through emails.

Reply:
I think we can still use the `breaking-change-x.0` label to communicate which flags are meant to be flipped for a major release. Added a sentence about that.

Reply:
`breaking-change-x.0` will probably only be useful for archiving purposes (i.e. history), since we only do one major release every 9+ months and it's basically impossible for two open `migration-ready` issues to be targeted for different major releases.

Reply:
That is true, but there could be incompatible flags that are not planned to be flipped for the next major release. For example, `--incompatible_disallow_empty_glob` and `--incompatible_strict_action_env`, they are ready for migration, but the migration might be very hard or blocked for other reason. By having this label, we can communicate which flags are actually going to be flipped in next major release, so that users can already start turning them on to prevent regression. 

/cc @aiuto @alexeagle WDYT? Do you still think a custom .bazelrc file is better? Can you parse this label to generate that bazelrc file?